src/components/tests/__snapshots__/QuickOpenModal.spec.js.snap

```diff
@@ -1458,14 +1458,14 @@ exports[`QuickOpenModal updateResults on enable 2`] = `
     >
       <Modal
         handleClose={[Function]}
-        status="entering"
+        status="exited"
```

Comment:
is this intended ?

Reply:
yeah - i'm scared of that too... i feel like we have that problem in the panel perhaps, so we should land this and then test that...

Reply:
i guess i could also test it now... before it lands :wink:

Reply:
yeah, i think it would be safer :)